### PR TITLE
Add DRA1 omit to regular normalization

### PR DESCRIPTION
**Pull request by tavinathanson**
To be used by the `mhctools` IEDB predictor.

### CODE REVIEW

File: mhcnames/normalization.py [hunk 1]
```diff
@@ -64,6 +64,13 @@ def normalize_allele_name(raw_allele):
     parsed_alleles = parse_classi_or_classii_allele_name(raw_allele)
     species = parsed_alleles[0].species
     normalized_list = [species]
+    # Optionally omit the alpha allele
+    if omit_dra1 and len(parsed_alleles) == 2:
+        alpha, beta = parsed_alleles
```
Comment (tavinathanson) [inline mhcnames/normalization.py]: Copied from the same code in `compact_allele_name` below.

File: mhcnames/normalization.py [hunk 2]
```diff
@@ -64,6 +64,13 @@ def normalize_allele_name(raw_allele):
...
+        # by convention the alpha allelle is omitted since it's assumed
+        # to be DRA1*01:01
+        if alpha == _DRA1_0101:
```
Comment (julia326) [inline mhcnames/normalization.py:78]: maybe bring the definition for `_DRA1_0101` up before this function?

File: mhcnames/normalization.py [hunk 3]
```diff
@@ -64,6 +64,13 @@ def normalize_allele_name(raw_allele):
     parsed_alleles = parse_classi_or_classii_allele_name(raw_allele)
     species = parsed_alleles[0].species
     normalized_list = [species]
+    # Optionally omit the alpha allele
```
Comment (julia326) [inline mhcnames/normalization.py]: worth saying why you might want to omit it, otherwise it'll be confusing since not in the spirit of "normalizing"

**Review by julia326 — COMMENTED** [review]
LGTM, couple quick things

File: mhcnames/normalization.py (hunk 2 above)
Comment (tavinathanson) [reply mhcnames/normalization.py:78]: Done

File: mhcnames/normalization.py (hunk 3 above)
Comment (tavinathanson) [reply mhcnames/normalization.py]: Done

**Review by julia326 — APPROVED** [review]

**Pull request merged**